README.adoc

```diff
@@ -12,3 +12,38 @@ The `xorg.conf` file should be placed in `/etc/X11/`.
 The `config.txt` file is the equivalent of a BIOS on the pi. It contains system configuration parameters. 
 This file is needed on the primary and secondary pi.
 The `config.txt` file should be placed in `/boot/`.
+
+= Instructions to run SetupScript
+
+Ensure to clone Repo to home directory (~) in order for the script to know where to move patch files upon running of the script.
+----
+cd ~
+git clone https://github.com/UCSolarCarTeam/Epsilon-Raspberry.git
+----
+navigate to Epsilon-Raspberry Directory
+----
+cd Epsilon-Raspberry
+----
+run the setupscript BUT make sure you are ready....to be ready:
```

Comment:
Reword this to:
```````
Before running the setup script, make sure that:
* You have a consistent blah blah
* You won't have to use the pi or move your computer until the script is complete. 
The script may take several hours to run, so be prepared to leave your computer on all night.
```````